src/grammar_tests.rs

```diff
@@ -116,7 +116,7 @@ fn test_parse_formula() {
             &mut fmla_vec.clone().into_iter(),
             &[wff, class],
             false,
-            &names,
+            &names as &Nameset,
```

Comment:
```suggestion
            &**names,
```
Alternatively and additionally, `Arc<T: Resolver>: Resolver` seems like a reasonable impl and would allow you to pass `&names` directly, although it is probably better at runtime to undo the indirection here rather than in the method calls.

Reply:
I'll check that - here this is only for test purpose, so we don't need to squeeze out the highest performance.

Reply:
See [Smart deref coercions instead of casts](https://github.com/david-a-wheeler/metamath-knife/pull/89/commits/586e0a36215a951c439b491470fcc1895c8501b3)